src/services/service.js

```diff
@@ -296,6 +296,31 @@ class Service {
     })
 
   }
+
+  handleInspect(projectId, dbType, col){
+    return new Promise((resolve, reject) => {
+      this.client.getJSON(`/v1/api/config/inspect/${projectId}/${dbType}/${col}`)
+      .then(({status, data}) => {
+        if(status !== 200){
+          reject("Internal server error")
```

Comment:
Return the error returned by the server `data.error` as other functions in service do